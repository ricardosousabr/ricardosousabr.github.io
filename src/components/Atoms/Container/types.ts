import React, { ReactNode } from "react";

export interface ContainerProps
  extends React.ButtonHTMLAttributes<HTMLButtonElement> {
  children: ReactNode;
}
