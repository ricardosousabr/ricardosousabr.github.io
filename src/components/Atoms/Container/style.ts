import styled from "styled-components";

export const ContainerStyle = styled.div`
  width: 100%;
  max-width: 1272px;
  padding: 0 20px;
  margin: 0 auto;
  box-sizing: border-box;
`