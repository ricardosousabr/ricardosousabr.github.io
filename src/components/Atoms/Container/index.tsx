import { ContainerProps } from "./types";
import { ContainerStyle } from "./style"

export default function Container({children}: ContainerProps) {
  return (
    <ContainerStyle>{children}</ContainerStyle>
  )
}