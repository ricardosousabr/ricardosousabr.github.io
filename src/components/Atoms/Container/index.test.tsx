import renderer from "react-test-renderer";
import Container from "./index";

test("Link renders correctly", () => {
  const tree = renderer.create(<Container>Click</Container>).toJSON();
  expect(tree).toMatchSnapshot();
});
