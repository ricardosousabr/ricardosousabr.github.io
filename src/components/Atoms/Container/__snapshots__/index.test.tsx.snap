// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link renders correctly 1`] = `
<div
  className="styles__ContainerStyle-sc-f8jz92-0 epgcrw"
>
  Click
</div>
`;
